src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptAsScriptingOperation.cs

```diff
@@ -548,18 +548,20 @@ private void SetScriptBehavior(ScriptingOptions options)
             }
         }
 
-        private static Dictionary<string, SqlServerVersion> LoadScriptCompatabilityMap()
+        private static Dictionary<string, SqlServerVersion> LoadScriptCompatibilityMap()
         {
-            Dictionary<string, SqlServerVersion> map = new Dictionary<string, SqlServerVersion>();
-            map.Add(SqlScriptOptions.ScriptCompatabilityOptions.Script140Compat.ToString(), SqlServerVersion.Version140);
-            map.Add(SqlScriptOptions.ScriptCompatabilityOptions.Script130Compat.ToString(), SqlServerVersion.Version130);
-            map.Add(SqlScriptOptions.ScriptCompatabilityOptions.Script120Compat.ToString(), SqlServerVersion.Version120);
-            map.Add(SqlScriptOptions.ScriptCompatabilityOptions.Script110Compat.ToString(), SqlServerVersion.Version110);
-            map.Add(SqlScriptOptions.ScriptCompatabilityOptions.Script105Compat.ToString(), SqlServerVersion.Version105);
-            map.Add(SqlScriptOptions.ScriptCompatabilityOptions.Script100Compat.ToString(), SqlServerVersion.Version100);
-            map.Add(SqlScriptOptions.ScriptCompatabilityOptions.Script90Compat.ToString(), SqlServerVersion.Version90);
-
-            return map;
+            return new Dictionary<string, SqlServerVersion>
+            {
+                {SqlScriptOptions.ScriptCompatibilityOptions.Script150Compat.ToString(), SqlServerVersion.Version150},
```

Comment:
Wait is this why scripting External Tables was failing in some cases?

Reply:
I don't think so, I just noticed 150 wasn't here.
Eric's funky external table that doesn't script also fails to script in SSMS, I haven't had time to look into it.

Reply:
Well that's better than "it fails in ADS only". Have we looped in the folks who added syntax support to target fixing that scripting scenario?